### PR TITLE
driver: crypto: hisilicon: update qm init configs

### PR DESCRIPTION
**Pull request by yuzexiyzx**
1. add qm_disable_clock_gate for HISI_QM_HW_V3
2. set doorbell timeout to QM_DB_TIMEOUT_SET ns

### CODE REVIEW

**Review by jenswi-linaro — COMMENTED** [review]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

File: core/drivers/crypto/hisilicon/hisi_qm.c [hunk 1]
```diff
@@ -639,6 +658,8 @@ void hisi_qm_dev_init(struct hisi_qm *qm)
 	/* Disable QM ras */
 	io_write32(qm->io_base + HISI_QM_ABNML_INT_MASK,
 		   HISI_QM_ABNML_INT_MASK_CFG);
+	/* set doorbell timeout to QM_DB_TIMEOUT_SET ns */
```
Comment (etienne-lms) [inline core/drivers/crypto/hisilicon/hisi_qm.c:661]: nitpicking: s/set/Set/ (for consistency)

File: core/drivers/crypto/hisilicon/hisi_qm.c [hunk 2]
```diff
@@ -617,11 +621,26 @@ enum hisi_drv_status hisi_qm_start(struct hisi_qm *qm)
...
+
+{
+	uint32_t val = 0;
+
+	if (qm->version == HISI_QM_HW_V2)
+		return;
+
+	val = io_read32(qm->io_base + QM_PM_CTRL0);
+	val |= QM_IDLE_DISABLE;
+	io_write32(qm->io_base + QM_PM_CTRL0, val);
```
Comment (etienne-lms) [inline core/drivers/crypto/hisilicon/hisi_qm.c:634]: `io_setbits32(qm->io_base + QM_PM_CTRL0, QM_IDLE_DISABLE);`?

**Comment by yuzexiyzx** [comment]
@etienne-lms All comments have been solved.

**Comment by yuzexiyzx** [comment]
@etienne-lms All comments have been solved.

**Comment by yuzexiyzx** [comment]
@jforissier All comments have been solved,but I can not reach @etienne-lms for a long time.

**Comment by etienne-lms** [comment]
`Acked-by: Etienne Carriere <etienne.carriere@foss.st.com>`
Sorry for the late feedback.

**Comment by yuzexiyzx** [comment]
Description has been modified.

**Comment by yuzexiyzx** [comment]
@jforissier All comments have been solved.

**Pull request merged**